js/everything.js

```diff
@@ -169,10 +169,32 @@ function cleanTime(time) {
     return ret;
 }
 
+function storeTime(time) {
+    var videoID = getCurrentVideoID();
+    if (window.sessionStorage && videoID) {
+        window.sessionStorage[videoID] = time;
+    }
+}
+
 function updatePlayerTime() {
-    $("#currentTime").text(cleanTime(player.getCurrentTime()));
+    var currentTime = player.getCurrentTime();
+    $("#currentTime").text(cleanTime(currentTime));
     // TODO: after the video loads, player.getDuration() may have changed +/- 1
```

Comment:
Can you remove the TODO: prefix on this comment, there's no action item there anymore :smile: